packs/equipment/howler-pistol.json

```diff
@@ -0,0 +1,87 @@
+{
+    "_id": "4cOXKB8NcL6JzGeI",
+    "img": "systems/pf2e/icons/default-icons/weapon.svg",
+    "name": "Howler Pistol",
+    "system": {
+        "baseItem": "dragon-mouth-pistol",
+        "bonus": {
+            "value": 0
+        },
+        "bonusDamage": {
+            "value": 0
+        },
+        "bulk": {
+            "value": 1
+        },
+        "category": "martial",
+        "containerId": null,
+        "damage": {
+            "damageType": "sonic",
+            "dice": 1,
+            "die": "d6"
+        },
+        "description": {
+            "value": "<p>Designed by a gunsmith as a personal challenge, a howler pistol strikes a miniature version of the sonic horn (see page 117) siege weapon's resonating orb with its hammer to generate beams of sonic energy. A howler pistol is a +1 striking dragon mouth pistol that deals sonic damage instead of piercing damage and doesn't have the concussive trait.</p>\n<p><strong>Activate—Resonant Shatter</strong><span class=\"action-glyph\">2</span> (manipulate)</p>\n<p><strong>Frequency</strong> once per day</p>\n<p><strong>Requirements</strong> The howler pistol is loaded;</p>\n<hr />\n<p><strong>Effect</strong> You over-torque the howler pistol's hammer to strike the resonating orb harder than normal. The gun fires a concentrated burst of sonic energy that shatters a nearby stone or rock surface, creating a @Template[type:burst|distance:10] at a point of impact within the gun's first range increment. Creatures in the area take @Damage[5d6[piercing]] damage (DC 27 basic Reflex save).</p>\n<hr />\n<p><strong>Craft Requirements</strong> The initial raw materials must include the powdered horn from a soniphak, processed into a resonating orb.</p>"
```

Comment:
```suggestion
            "value": "<p>Designed by a gunsmith as a personal challenge, a howler pistol strikes a miniature version of the sonic horn (see page 117) siege weapon's resonating orb with its hammer to generate beams of sonic energy. A howler pistol is a +1 striking dragon mouth pistol that deals sonic damage instead of piercing damage and doesn't have the concussive trait.</p>\n<p><strong>Activate—Resonant Shatter</strong><span class=\"action-glyph\">2</span> (manipulate)</p>\n<p><strong>Frequency</strong> once per day</p>\n<p><strong>Requirements</strong> The howler pistol is loaded;</p>\n<hr />\n<p><strong>Effect</strong> You over-torque the howler pistol's hammer to strike the resonating orb harder than normal. The gun fires a concentrated burst of sonic energy that shatters a nearby stone or rock surface, creating a @Template[type:burst|distance:10] at a point of impact within the gun's first range increment. Creatures in the area take @Damage[5d6[piercing]] damage (@Check[type:reflex|dc:27|basic:true] save).</p>\n<hr />\n<p><strong>Craft Requirements</strong> The initial raw materials must include the powdered horn from a soniphak, processed into a resonating orb.</p>"
```